ods_ci/tests/Tests/0100__platform/0101__deploy/0101__installation/0108__operator.robot

```diff
@@ -4,11 +4,66 @@ Documentation       Post install test cases that verify OCP operator resources a
 Library           OpenShiftLibrary
 Resource          ../../../../Resources/ODS.robot
 Resource          ../../../../Resources/RHOSi.resource
+Resource          ../../../../Resources/OCP.resource
+Resource          ../../../../Resources/Common.robot
+Resource          ../../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
 Suite Setup       RHOSi Setup
 Suite Teardown    RHOSi Teardown
 
 
+*** Variables ***
+${ODH_RELEASE_NAME}=     Open Data Hub
+${RHOAI_SELFMANAGED_RELEASE_NAME}=      OpenShift AI Self-Managed
+${RHOAI_MANAGED_RELEASE_NAME}=      OpenShift AI Cloud Service
```

Comment:
Can we put these in a centralized place like Common.robot or similar? Otherwise we duplicate these "constants" and we have to change them in multiple places if their values change.

Reply:
yeah, we can do that